man/git-magic.md

```diff
@@ -0,0 +1,53 @@
+git-magic(1) -- commits with generated message
+================================
+
+## SYNOPSIS
+
+`git-magic` [-a] [OPTIONS]
+
+## DESCRIPTION
+
+Stages everything (optional), with prompting user to confirm.
+Produces summary of changes for commit message from `git status --porcelain` output.
+Commits staged changes with generated commit message.
+Opens editor to modify generated commit message. (optional)
+
+## OPTIONS
+
+`-a` option adds everything including untracked files.
```

Comment:
Please doc each option in the format (each option has its own line and line break between options): https://github.com/tj/git-extras/blob/f6fa554077c9ce574ae2ea66fa7462d18f2cc7b6/man/git-summary.md#L37-L45

Reply:
Updated documentation accordingly.